code/modules/surgery/organs/augments_arms.dm

```diff
@@ -393,6 +393,16 @@
 	zone = BODY_ZONE_L_ARM
 	syndicate_implant = TRUE
 
+/obj/item/organ/cyberimp/arm/syndie_hammer
+	name = "Vxtvul Hammer implant"
+	desc = "A folded Vxtvul Hammer designed to be incorporated into preterni chassis. Surgery can permit it to fit in other organic bodies."
+	contents = newlist(/obj/item/twohanded/vxtvulhammer)
+	syndicate_implant = TRUE
+
+/obj/item/organ/cyberimp/arm/syndie_hammer/l
+	zone = BODY_ZONE_L_ARM
+	syndicate_implant = TRUE //Just to be safe, everything tells me it's not needed but I don't trust implant code
```

Comment:
you setting it again would not change anything. if its set by parent its automatically inherited, theres no reason why implant code would break that, and if it did, your solution wouldnt work either.